src/components/Calendar.vue

```diff
@@ -1,13 +1,45 @@
 <template>
-  <div>
-    <h1>Calendar View</h1>
-  </div>
+  <v-container fluid>
+    <full-calendar :events="fcEvents"/>
+  </v-container>
 </template>
 
 <script>
-export default {
 
+export default {
+  components: {
+    'full-calendar': require('vue-fullcalendar')
+  },
+  computed: {
+    fcEvents () {
+      const Slides = this.$store.getters.getSlide
+      const filteredSlides = (Slides.map((slide) => {
+        const newSlide = {}
+        if (slide &&
+            slide.title &&
+            slide.title.content &&
+            slide.date &&
+            slide.date.content) {
+          // Copy the title and start time
+          newSlide.title = slide.title.content
+          newSlide.start = this.addOneDay(slide)
+        }
+        return newSlide
+      }))
+      return filteredSlides
```

Comment:
also instead of constructing value and then returning, why not just return the map() so the function would be `return this.$store.getters.allSlides.map(slide => { .... })`

Reply:
Done